src/elements/content-preview/ContentPreview.js

```diff
@@ -384,6 +388,15 @@ class ContentPreview extends React.PureComponent<Props, State> {
         if (haveExperiencesChanged && this.preview && this.preview.updateExperiences) {
             this.preview.updateExperiences(previewExperiences);
         }
+
+        if (advancedContentInsights) {
+            const { advancedContentInsights: prevContentInsightsOptions } = prevProps;
+            const haveContentInsightsChanged = prevContentInsightsOptions !== advancedContentInsights;
```

Comment:
I'm not entirely sure but would `haveContentInsightsChanged` always be true? since they're objects, it would compare references and so would always be different right?

Reply:
@tjuanitas you are right. I was comparing before only the isActive prop, but I changed it and didn't do the deep comparison...My bad! But I think it seems we have the same issue with haveExperiencesChanged in this file. Should I do something?

Reply:
@marcoartaviaq nice catch!

I'm discussing with @megansmith-box and getting her opinion on it. it sounds logical that if some object from `prevProps` is unchanged, then it would use the same object reference. although from testing it, that doesn't seem to be the case so the `haveExperiencesChanged` calculation could've been an oversight.

I would leave `haveExperiencesChanged` as-is for now just in case and we can address it in a separate PR